network/builder/src/builder.rs

```diff
@@ -184,96 +179,97 @@ impl NetworkBuilder {
     }
 
     pub fn create(
-            chain_id: &ChainId,
-            role: RoleType,
-            config: &mut NetworkConfig,
-            libra_db: Arc<dyn DbReader>,
-            waypoint: Waypoint,
-        ) -> (Runtime, NetworkBuilder) {
-            let runtime = Builder::new()
-                .thread_name("network-")
-                .threaded_scheduler()
-                .enable_all()
-                .build()
-                .expect("Failed to start runtime. Won't be able to start networking.");
-
-            let identity_key = config.identity_key();
-            let peer_id = config.peer_id();
-
-            let mut network_builder = NetworkBuilder::new(
-                runtime.handle().clone(),
-                chain_id.clone(),
-                config.network_id.clone(),
-                role,
-                peer_id,
-                config.listen_address.clone(),
+        chain_id: &ChainId,
```

Comment:
it looks like you had some tabbing issue in this PR and it isn't clear what's changed, is this something you could fix?